Maddori.Apple/Maddori.Apple/Global/UIComponent/View/FeedbackTypeLabelView.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  FeedbackTypeLabelView.swift
+//  Maddori.Apple
+//
+//  Created by 김유나 on 2022/11/02.
+//
+
+import UIKit
+
+import SnapKit
+
+final class FeedbackTypeLabelView: UILabel {
+    private enum Size {
+        static let width: CGFloat = 158
+        static let height: CGFloat = 46
+        static let labelPadding: CGFloat = UIScreen.main.bounds.width - SizeLiteral.leadingTrailingPadding * 2 - Size.width * 2
+    }
+    var feedbackType: FeedBackType = .continueType {
+        didSet { setupAttribute(feedbackType) }
+    }
+    
+    // MARK: - property
+    
+    private let continueShadowView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white100
+        view.layer.shadowOpacity = 0.2
+        view.layer.shadowOffset = CGSize(width: 0, height: 0)
+        view.layer.cornerRadius = SizeLiteral.componentCornerRadius
+        return view
+    }()
+    private let stopShadowView: UIView = {
+        let view = UIView()
+        view.backgroundColor = .white100
+        view.layer.shadowOpacity = 0.2
+        view.layer.shadowOffset = CGSize(width: 0, height: 0)
+        view.layer.cornerRadius = SizeLiteral.componentCornerRadius
+        return view
+    }()
+    private lazy var continueLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Continue"
+        label.font = .main
+        label.textAlignment = .center
+        label.clipsToBounds = true
+        label.layer.cornerRadius = SizeLiteral.componentCornerRadius
+        return label
+    }()
+    private lazy var stopLabel: UILabel = {
+        let label = UILabel()
+        label.text = "Stop"
+        label.font = .main
+        label.textAlignment = .center
+        label.clipsToBounds = true
+        label.layer.cornerRadius = SizeLiteral.componentCornerRadius
+        return label
+    }()
+    
+    // MARK: - life cycle
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        render()
+    }
+    
+    required init?(coder: NSCoder) { nil }
+    
+    private func render() {
+        self.addSubview(continueShadowView)
+        continueShadowView.snp.makeConstraints {
+            $0.width.equalTo(Size.width)
+            $0.height.equalTo(Size.height)
+            $0.leading.top.equalToSuperview()
+        }
+        
+        continueShadowView.addSubview(continueLabel)
+        continueLabel.snp.makeConstraints {
+            $0.width.equalTo(Size.width)
+            $0.height.equalTo(Size.height)
+            $0.leading.equalTo(continueShadowView.snp.leading)
+            $0.top.equalTo(continueShadowView.snp.top)
```

Comment:
p4;

```suggestion
            $0.edges.equalToSuperview()
```
슈퍼뷰의 크기와 맞추는 거라면 요로코롬 바꿀 수 있을 것 같아요 !

Reply:
[efb88f9](https://github.com/DeveloperAcademy-POSTECH/MacC-Team-Maddori.Apple/pull/68/commits/efb88f9bfbb399b9c75c923c6bfcfc1c97b7f540)
와우 그러네요! 감사합니다 ! 코드 줄어서 넘 편안하네요,,,😺